app/products/calls/components/call_message/call_message.tsx

```diff
@@ -32,9 +32,10 @@ type CallMessageProps = {
     currentChannelName: string;
     callChannelName: string;
     intl: typeof IntlShape;
+    isCloudLimitRestricted: boolean;
 }
 
-const getStyleSheet = makeStyleSheetFromTheme((theme: Theme) => {
+const getStyleSheet = makeStyleSheetFromTheme(({theme, isCloudLimitRestricted}: CallMessageProps) => {
```

Comment:
this should not be done this way..

two things happening here:
1. is never memoizing the styles as it receives a new object every render
2. the purpose for this is to just to use it with the theme

What to do?
Change it so that it only uses the `theme` as an argument, create styles for cloudRestricted, and then in the component apply the right style based on the `isCloudLimitRestricted` prop.

Same with the other components.

Reply:
@enahum I was following what you did here: https://github.com/mattermost/mattermost-mobile/blob/4c8594d330782e6915cccee0740429a96cef3e17/app/screens/gallery/gallery_file/gallery_file.tsx#L25

But sure, I can change it. 👍

Reply:
Well, what I did there is totally wrong as well :P